interp/cfg.go

```diff
@@ -975,14 +975,15 @@ func (interp *Interpreter) cfg(root *node, importPath string) ([]*node, error) {
 					vtyp := &itype{cat: valueT, rtype: rtype.Elem()}
 					err = check.mapLitExpr(child, ktyp, vtyp)
 				}
+				// TODO(mpl): how come we don't have anything to do for reflect.Ptr here?
```

Comment:
Nothing more to do here, as a composite expression always applies directly to the target type. A pointer indirection, if any,  is handled in the parent AST node. At runtime (run.go) of course the pointer indirection must be handled.

Reply:
you can safely remove this comment

Reply:
you can safely remove this comment

Reply:
done